unique_toolkit/pyproject.toml

```diff
@@ -20,7 +20,7 @@ numpy = "^1.26.4"
 python-dotenv = "^1.0.1"
 regex = "^2024.5.15"
 tiktoken = "^0.7.0"
-unique-sdk = "^0.9.6"
+unique-sdk = "^0.9.7"
```

Comment:
Please replace this import with the one below `unique-sdk = {path = "../unique_sdk"}`. We don't need them twice.